### PR TITLE
made task 'moyo header'

### PR DESCRIPTION
**Pull request by TKuts**
- [DEMO LINK](https://TKuts.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://TKuts.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,12 +3,46 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </div>
+      <nav class="navigation">
```
Comment (SerhiiKirik) [inline src/index.html:17]:
```suggestion
      </div>
      
      <nav class="navigation">
```

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,46 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
```
Comment (SerhiiKirik) [inline src/index.html:28]: Do it for more readable
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">Smartphones</a>
          </li>
          
          <li class="nav__item">
```

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,46 @@
...
+            <a href="#" class="nav__link" >Laptops & Computers</a>
```
Comment (SerhiiKirik) [inline src/index.html:29]:
```suggestion
            <a href="#" class="nav__link">Laptops & Computers</a>
```

File: src/index.html [hunk 4]
```diff
@@ -3,12 +3,46 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item ">
```
Comment (SerhiiKirik) [inline src/index.html:40]:
```suggestion
          <li class="nav__item">
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --light-blue: rgb(0, 172, 220);
+  --white: rgb(255, 255, 255);
+}
+
+* {
```
Comment (SerhiiKirik) [inline src/style.css:8]: These classes should be in the `html` selector
```suggestion
html {
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,78 @@
...
+
+.is-active {
+  color: var(--light-blue);
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  border-radius: 8px;
+  left: 0;
```
Comment (SerhiiKirik) [inline src/style.css:73]: Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🔥 
Let's deploy your tests 🧑‍💻 
image

**Comment by TKuts** [comment]
Fixed the errors and used this link (htmlAcademy) to style the code

- [DEMO LINK](https://TKuts.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://TKuts.github.io/layout_moyo-header/report/html_report/)

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Fix tests please

add `data-cy` attributes where it needed

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,83 @@
...
+
 body {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0;
+}
+
+header {
```
Comment (Anastasiia-Svintsova) [inline src/style.css:22]: Use classes instead 
image

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- The header should have a full width according to the mockup. The spaces between the header and screen edges should be the same for all screens.
image

File: src/index.html [hunk 8]
```diff
@@ -3,12 +3,58 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```
Comment (Anastasiia-Svintsova) [inline src/index.html:36]: Fix the code style 
```suggestion
            <a 
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Check my comments, please

**Comment by TKuts** [comment]
> eader {

image

I disagree with this comment, because the layout clearly shows that the content is smaller. And the basic 1200px is already with content and indents.

I corrected the rest of the comments.

File: src/index.html [hunk 9]
```diff
@@ -3,12 +3,95 @@
...
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
+
 </html>
```
Comment (polosanya) [inline src/index.html:97]:
```suggestion
  </body>
</html>
```
Don't add empty lines between parent and child elements

**Review by polosanya — APPROVED** [review]
Nice!

**Pull request closed without merging**